Comanche055/ASSEMBLY_AND_OPERATION_INFORMATION.agc

```diff
@@ -865,15 +860,15 @@
 # -VV-
 # POSITION8		XXXX.X NAUT MI		DP	BIT 1 OF LOW REGISTER =
 #									 -28
-#							4 x 6,373,338 x 2
+#							4 X 6,373,338 X 2
 #							--------------------
-#								1852
+#							      1852
 #							NAUT MI.
 #
 # -XX-
-# POSITION 9		XXX.XX NAUT MI		DP	BIT 1 OF LOW REGISTER =
```

Comment:
@chrislgarry It might have happened during their validation process. Read here: http://www.ibiblio.org/apollo/Colossus.html#ValidityApollo11

You also asked about the unit change itself. It might have to do with one of the extended verbs (verb 67), which in earlier AGC versions displayed nautical miles and in Comanche055 it was changed to feet. Not sure though.